e2e/ts_library/WORKSPACE

```diff
@@ -26,6 +26,14 @@ load("@build_bazel_rules_nodejs//:defs.bzl", "yarn_install")
 
 yarn_install(
     name = "npm",
+    manual_build_file_contents = """
+filegroup(
+  name = "node_modules_filegroup",
```

Comment:
add a comment here that this is to exclude hammerjs because the test asserts that it's missing?